Cargo.toml

```diff
@@ -24,15 +24,15 @@ chameleon = "0.1.0"
 scale-info = { version = "1.0.0", features = ["bit-vec"] }
 futures = "0.3.13"
 hex = "0.4.3"
-jsonrpsee = { version = "0.7.0", features = ["macros", "ws-client", "http-client", "client-ws-transport"] }
+jsonrpsee = { version = "0.8.0", features = ["macros", "ws-client", "http-client", "client-ws-transport"] }
```

Comment:
```suggestion
jsonrpsee = { version = "0.8.0", features = ["ws-client", "http-client"] }
```

Reply:
The `feature macros` is unused and `client-ws-transport` is enabled by the `ws-client`.